build.sbt

```diff
@@ -42,24 +42,23 @@ lazy val server = (project in file("server"))
       (baseDirectory.value / "test" / "tests" * "*").get map
         (x => x -> ("test/tests/" + x.getName)),
 
+    resourceDirectory in Test := baseDirectory.value / "test-resources",
```

Comment:
Do we need to create additional folder in the middle: "test / scala / test" as per standard maven template here?